WPFSKillTree/SkillTreeFiles/Compute.cs

```diff
@@ -2310,7 +2310,7 @@ public static void Initialize(SkillTree skillTree, ItemAttributes itemAttrs)
 
             CoreAttributes();
 
-            Implicit = new AttributeSet(skillTree.ImplicitAttributes(Global));
+            Implicit = new AttributeSet(SkillTree.ImplicitAttributes(Global, skillTree.Level));
```

Comment:
OK. But in the original code, the property used was SkillTree._level, it worked exactly same way as now.

Will it be better to add bounds checking to property SkillTree.Level to make it work everywhere or change it just in Compute.cs ?